presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java

```diff
@@ -23,6 +23,8 @@
 import com.facebook.presto.sql.tree.ComparisonExpression;
```

Comment:
you could make `is (not) null` upper case in the commit message. IMO that way it would be more readable as it would point to SQL. See:
```
Add stats calculation for FilterNode IS (NOT) NULL expression

```